### ISSUE TITLE
Specify non-standard paths to game files

### ISSUE THREAD

**Issue by rsubtil** [retrohub-org/retrohub#190]
Initially the idea was to centralize all game file locations, in the same way ES does. However, the community has been suggesting allowing to select individual folders per system, which also solves another common pain point for users of other front-ends which is "forced" system folders, and the inability to seamlessly use their library on different applications.

**Comment by Ro-Borges** [comment]
I agree with this. Instead of forcing folde structures for games, emulators and media assets, we should be able to specify the folders per system to be able to use our library with different aplications
